Makefile

```diff
@@ -0,0 +1,11 @@
+provision:
+	./tools/laptop_setup.sh
+
+hugo-start:
+	cd docs && hugo server --gc --disableFastRender > /dev/null 2>&1 &
+	
+challenge-reset:
```

Comment:
Worth adding a `challenge-reset-quick` maybe? 
Most challenges won't need a vagrant destroy + up. Simply the repo refresh would be enough. Just a time saving measure.